app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageController.scala

```diff
@@ -592,7 +600,8 @@ class SingleDisposalsTriageController @Inject() (
   private def updateDisposalDate(
     d: LocalDate,
     taxYear: Option[TaxYear],
-    answers: SingleDisposalTriageAnswers
+    answers: SingleDisposalTriageAnswers,
```

Comment:
in `updateDraftReturnForDisposalDate` gainOrLossAfterReliefs needs to be unset as well